packages/psss/src/components/Layout.js

```diff
@@ -22,15 +22,16 @@ export const Container = styled(MuiContainer)`
   }
 `;
 
-const headerHeight = '355px';
+const headerHeight = '315px';
+const footerHeight = '48px';
 
 /*
  * Main section that holds the data table
  */
 export const Main = styled.main`
   padding-top: 1rem;
   padding-bottom: 5rem;
-  min-height: calc(100vh - ${headerHeight});
+  min-height: calc(100vh - ${headerHeight} - 1rem - 5rem + ${footerHeight});
```

Comment:
Good call splitting the padding subtraction calculation instead of using `-6rem`, to make it more obvious 👍